src/ansys/simai/core/data/projects.py

```diff
@@ -126,7 +135,7 @@ def is_trainable(self) -> bool:
         tt = self._client._api.is_project_trainable(self.id)
         return IsTrainableInfo(**tt)
 
-    def get_variables(self) -> Optional[Dict[str, List[str]]]:
+    def get_variables(self) -> Optional[dict[str, List[str]]]:
```

Comment:
I think you might have to rebase for the tests to work properly, if I remember correctly this isn't valid in Python 3.9 which we're supposed to support (the typing part).

Reply:
I think I corrected it.